resources/manage.rb

```diff
@@ -23,7 +23,7 @@
 property :users, Array, description: 'Array of Hashes that contains all the users that you want to create with the users cookbook.', default: []
 property :cookbook, String, description: 'name of the cookbook that the authorized_keys template should be found in.', default: 'users'
 property :manage_nfs_home_dirs, [true, false], description: 'specifies if home_dirs should be managed when they are located on a NFS share.', default: true
-
+property :system, [true, false], description: 'specefies if the group should be a system group. See the -r option of groupadd', default: false
```

Comment:
FWIW you need to also fix the typo in the line above that was missed.
```suggestion
property :system, [true, false], description: 'specifies if the group should be a system group. See the -r option of groupadd', default: false
```